shuffle/cli/src/account.rs

```diff
@@ -29,24 +31,18 @@ use std::{
 };
 
 // Creates new account from randomly generated private/public key pair.
-pub fn handle(home: &Home, root: Option<PathBuf>) -> Result<()> {
-    if !home.get_shuffle_path().is_dir() {
-        return Err(anyhow!(
-            "A node hasn't been created yet! Run shuffle node first"
-        ));
-    }
+pub fn handle(home: &Home, root: Option<PathBuf>, network: Network) -> Result<()> {
+    let network_home = NetworkHome::new(get_shuffle_networks_path().as_path(), network.get_name());
+    check_node_deployed_if_localhost_used(home, &network)?;
 
-    if home.get_latest_path().exists() {
-        let wants_another_key = confirm_user_decision(home);
-        if wants_another_key {
-            let time = duration_since_epoch();
-            let archive_dir = home.create_archive_dir(time)?;
-            home.archive_old_key(&archive_dir)?;
-            home.archive_old_address(&archive_dir)?;
-        } else {
-            return Ok(());
+    if network_home.get_latest_path().exists() {
```

Comment:
Can we rename this to `get_latest_account_path` or wtever latest thing we're getting?